openstudiocore/src/cli/CMakeLists.txt

```diff
@@ -281,12 +281,10 @@ target_link_libraries(openstudio
 )
 
 if (UNIX AND NOT APPLE)
-  target_link_libraries(openstudio ${RUBY_STATIC_LIBRARY} ${EXTENSION_LIBS} dl crypt ssl)
+  target_link_libraries(openstudio ${RUBY_STATIC_LIBRARY} ${EXTENSION_LIBS} dl crypt)
```

Comment:
We will want the CLI to have access to ssl libraries as we want to use the embedded ruby to connect to websites over https